Library/Homebrew/cmd/fetch.rb

```diff
@@ -41,6 +41,18 @@ def fetch
     end
   end
 
+  # For Homebrew's internal use.
+  # Fetches a bottle for any arch, without falling back to source download.
+  def fetch_bottle_any_arch(names)
```

Comment:
I think we are trying to eliminate cross commands `require` instead of the opposite. Put it in `cmd/pull` would be better if you needed.